utility/types/proto/actor.proto

```diff
@@ -0,0 +1,30 @@
+syntax = "proto3";
+package utility;
+
+option go_package = "github.com/pokt-network/pocket/utility/types";
+
+enum Pool_Names {
+  DAO = 0;
```

Comment:
Please make the `0th` value value `PoolNameUnspecified`. Same thing for `ActorType` below. We're already doing it for `StakeStatus`.

This is a best practice that all enums at Google use (and is a best practice per https://developers.google.com/protocol-buffers/docs/proto3#enum) because the 0th value is the default.

If someone, somewhere, doesn't specify the value explicitly, it defaults to the 0th enum and therefore helps capture errors early.